### DISCUSSION TITLE
Cannot install extension on Supabase admin dashboard

### DISCUSSION THREAD

**Discussion by silverfoxlabs** [supabase/supabase#17709] [Questions] [answered]
If I try to toggle supabaseutils it errors out

```
Failed to toggle SUPAUTILS: failed to create pg.extensions: extension "supautils" has no installation script nor update path for version "0.1.0"
```

**Comment by GaryAustin1** [comment]
I can confirm I get the same error.   But I don't know anything about its status.

**Comment by steve-chavez** [comment, marked as answer]
Check https://github.com/supabase/supautils/issues/62

> there's no reason for end users to ever enable this extension
> This is also fixed on master https://github.com/supabase/supautils/commit/b032becdd10693aac8a9b625d4db7e7430db7b52. It will get cleared on a new release/deploy.
